### ISSUE TITLE
fix version names in docs

### ISSUE THREAD

**Issue by JamesSaxon** [pysal/access#46]
@jGaboardi, ever since the update to versioneer (#39), I noticed that the versions listed in the docs show up weird -- like `0+untagged.1.g109a527`:

<img width="408" alt="image" src="https://user-images.githubusercontent.com/12751805/231605291-589d62ce-99eb-423c-92d6-f31a17042288.png">

Could you either advise how to fix this with a new tag, or do so and mention me?

**Comment by jGaboardi** [comment]
Seems like no initial tag was ever made. I would suggest `v0.0.1`. When you are ready to create the tag from you machine do:

```
git tag -a v0.0.1 -m v0.0.1
```
then
```
git push upstream v0.0.1
```

**Comment by JamesSaxon** [comment]
OK, thanks. I think we were at 1.1.8 on the old naming via the py files, so I'll start at 1.1.9.

**Comment by JamesSaxon** [comment]
Hey @jGaboardi, the release build failed -- 

https://github.com/pysal/access/actions/runs/4683952297/jobs/8299566391

It seems like versioneer was set up for `master` whereas the repo was switched over to `main`?

Is it as simple as find/replacing `s/master/main/g/` in `versioneer.py`?

**Comment by jGaboardi** [comment]
No, the failure is coming from the [`tools/gitcount.ipynb`](https://github.com/pysal/access/actions/runs/4683952297/jobs/8299566391#step:7:1) notebook. This is [failing](https://github.com/pysal/access/actions/runs/4683952297/jobs/8299566391#step:7:74) because the first line of [`__init__`](https://github.com/pysal/access/blob/main/access/__init__.py) no longer contains the version. The [solution I used previously](https://github.com/pysal/spaghetti/pull/654/files#diff-ab45a5565661ce145449ee7e61410c0e3e745bcf44a7b786802b78fd8b7f786dR153-R170) was to replace the `# get __version__` cell with:
```python
# get __version__
%run ../access/_version.py
```
then add another cell
```python
__version__ = get_versions()["version"]
```

**Comment by JamesSaxon** [comment]
Hey @jGaboardi, I noticed that we still have the weird versions in the [docs](https://pysal.org/access/index.html) -- 

<img width="353" alt="image" src="https://github.com/pysal/access/assets/12751805/0eefca38-2912-41a4-be82-d1506701811e">


I had pushed a tag long ago, and #47 fixed the other issue noted above.  

It looks like the docs build was 'fine" at that time -- 

https://github.com/pysal/access/actions/runs/4909247529

I just tried making a release from that v1.1.9 tag, but perhaps I need to do something else to trigger the build?

I'm not familiar with versioneer or the current doc setup; do you have other ideas of what to check?

**Comment by jGaboardi** [comment]
I will give it attention over the weekend, if that's OK.  Across the ecosystem we have been a bit fluid recently with adopting improved infrastructure. For example, swaping out `setup.py` for `pyproject.toml`, `versioneer` for `setuptools_scm`, etc. Right now @weikang9009 and I are in the process of [modernizing `giddy`](https://github.com/pysal/giddy/issues/200) in this fashion; including updating docs actions.

**Comment by jGaboardi** [comment]
xref #21
